.vscodeignore

```diff
@@ -6,3 +6,12 @@
 src/**
 out/test/**
 **/*.js.map
+SampleApplication/
+SampleExponentApplication/
```

Comment:
The samples are intended for development, e.g. testing out features / running tests, rather than good examples of creating an app. Since it is intended for development only, and not useful for the end-user, we shouldn't be packaging it with the extension.